front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -2,31 +2,20 @@ import React, { useState } from 'react';
 
 import { useSelector } from 'react-redux';
 
-import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
-import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
+import useStdcm from 'applications/stdcm/hooks/useStdcm';
 import { useOsrdConfSelectors } from 'common/osrdContext';
-import { useStoreDataForSpeedLimitByTagSelector } from 'common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector';
-import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
-import { getTrainScheduleV2Activated } from 'reducers/user/userSelectors';
 
 import StdcmConfig from '../components/StdcmConfig';
 import StdcmHeader from '../components/StdcmHeader';
-import StdcmResults from '../components/StdcmResults';
+import StdcmResults from '../components/StdcmResults/index';
```

Comment:
Index is the default file, you can remove it ;) 
```suggestion
import StdcmResults from '../components/StdcmResults';
```

Reply:
fix